packages/verification-vue-component/src/components/Verification/item/cards/CertifyingCard.vue

```diff
@@ -68,7 +68,7 @@
                     <DataPanel :key="registerEvents[0].ref"
                                :icon-src="(registerEvents[0].scope !== 'certify') ? iconUser : null"
                                :md-icon="(registerEvents[0].scope === 'certify') ? mdiDomain : null"
-                               :title="issuerDisplayName(registerEvents[0])">
```

Comment:
I think "issuerDisplayName" isn't used anymore in this file and can be removed from "inject"

Reply:
Good catch, removed it.